package.json

```diff
@@ -67,7 +67,7 @@
     "gulp-clean": "^0.3.2",
     "gulp-concat": "^2.6.0",
     "gulp-connect": "^5.7.0",
-    "gulp-eslint": "^4.0.0",
+    "gulp-eslint": "^6.0.0",
```

Comment:
@musikele we were seeing an error during the linting step when we used the `??` operator. 
After upgrading `gulp-eslint` the error is gone. Is it ok to upgrade?

Reply:
I am asking around. Will respond ASAP

Reply:
Hi @leonelcuevas , We have committed the gulp-eslint version change separately, can you rebase your PR ? then we can accept it

Reply:
done! thank you!